tools/webpack-plugin/src/BacktracePluginV4.ts

```diff
@@ -1,17 +1,22 @@
-import { ContentAppender, DebugIdGenerator } from '@backtrace/sourcemap-tools';
+import { ContentAppender, DebugIdGenerator, SourceMapUploader } from '@backtrace/sourcemap-tools';
 import crypto from 'crypto';
+import path from 'path';
 import { Compiler, WebpackPluginInstance } from 'webpack';
 import { BacktraceWebpackSourceGenerator } from './BacktraceWebpackSourceGenerator';
 import { BacktracePluginOptions } from './models/BacktracePluginOptions';
 
 export class BacktracePluginV4 implements WebpackPluginInstance {
     private readonly _sourceGenerator: BacktraceWebpackSourceGenerator;
+    private readonly _sourceMapUploader?: SourceMapUploader;
 
     constructor(public readonly options?: BacktracePluginOptions) {
         this._sourceGenerator = new BacktraceWebpackSourceGenerator(
             options?.debugIdGenerator ?? new DebugIdGenerator(),
             new ContentAppender(),
         );
+
+        this._sourceMapUploader =
```

Comment:
should we do anything if source map uploader is undefined? Is there any point of doing anything without it?